contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/HadoopStoreBuilder.java

```diff
@@ -220,7 +220,11 @@ public void build() {
             FileSystem tempFs = tempDir.getFileSystem(conf);
             tempFs.delete(tempDir, true);
 
-            long size = sizeOfPath(tempFs, inputPath);
+            //use a list in case we pass a csv path list
```

Comment:
Is it supported to pass a CSV path list? (I didn't think so, but earlier today you taught me something else which I didn't think Voldemort supported, so who knows eh!)